scara_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50 


  # Velocity Controllers ------------------------------------
  joint1_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  
  joint2_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint2
    pid: {p: 100.0, i: 0.01, d: 10.0} 

  joint4_velocity_controller:
    type:  effort_controllers/JointVelocityController
    joint: joint4
    pid: {p: 100.0, i: 0.01, d: 10.0} 

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 100.0, i: 0.01, d: 1000.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 50.0, i: 0.01, d: 200.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint4
    pid: {p: 60.0, i: 0.01, d: 180.0}